### ISSUE TITLE
Campus specific information missing page if not logged in

### ISSUE THREAD

**Issue by cpwillett** [CDLUC3/dash#85]
It looks like the logo doesn't appears if you're not logged in:
![ucr-contactus-notloggedin](https://cloud.githubusercontent.com/assets/1558172/12311382/9af69e1e-ba0b-11e5-8bff-d3308b5863d3.png)

But does appear if you are logged in:
![ucr-contactus-loggedin](https://cloud.githubusercontent.com/assets/1558172/12311371/907e01d4-ba0b-11e5-891a-6b663472d8e2.png)

**Comment by marisastrong** [comment]
https://www.pivotaltracker.com/story/show/111652598

**Comment by cpwillett** [comment]
This is true for all campus instances. Other customizations are missing from this page when not logged in as well: navbar link to campus instance and footer customizations.
